exporter/importables.go

```diff
@@ -845,4 +847,70 @@ var resourcesMap map[string]importable = map[string]importable{
 			return nil
 		},
 	},
+	"databricks_workspace_conf": {
+		Service: "workspace",
+		Name: func(d *schema.ResourceData) string {
+			return globalWorkspaceConfName
+		},
+		Import: func(ic *importContext, r *resource) error {
+			wsConfAPI := workspace.NewWorkspaceConfAPI(ic.Context, ic.Client)
+			keys := map[string]interface{}{"enableIpAccessLists": false}
```

Comment:
this is very limited wsconf exporter. add a way to add more fields there.

Reply:
I didn't want to go with more than officially documented

Reply:
https://registry.terraform.io/providers/databrickslabs/databricks/latest/docs/resources/workspace_conf there are few more ;)

Reply:
added